src/main/java/org/folio/circulation/domain/NoteLink.java

```diff
@@ -0,0 +1,19 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+
+public class NoteLink extends JsonObject {
```

Comment:
What is a note link? And why does circulation care about it?

Reply:
The note link is used by a note to attribute it to a particular entity in the system. So for the purposes of this issue a note would need a NoteLink where the id is the UUID of the user and the type is the string "users". A note cannot be created without a link, and that is the reason why circulation needs it.